src/types/artist.ts

```diff
@@ -16,11 +21,13 @@ export type ArtistProfile = TimeField & {
   websiteUrl?: string;
 }
 
-export const mapArtist = (artistProfile: ArtistProfile): Artist => {
+export const mapArtist = (artistProfile: ArtistProfile): Partial<Artist> => {
```

Comment:
hmm should this be Partial? shouldn't we expect a full/complete artist object always?